thunder/executors/nvfuserex_impl.py

```diff
@@ -544,6 +559,12 @@ def create_fusion_definition_wrapper(
     store_inputs: None | bool = get_compile_option(
         "nv_store_fusion_inputs", "Allow nvFuser to store fusion inputs for repro."
     )
+    _enable_options: None | list[str] = get_compile_option(
```

Comment:
nitpick: I think just `enable_options` should be fine. same thing for `disable_options`

Reply:
Sure. For values, this naming is fine too. We do use `_enable_options` for teh argument names in `fd.execute`

Reply:
Done!